schema.prisma

```diff
@@ -15,6 +15,7 @@ model Subscription {
   nostrWalletConnectUrl String /// @encrypted
   recipientLightningAddress String
   amount Int
+  currency String?
```

Comment:
This changes what `amount` is e.g. for analytics.

Originally we could sum the amount, this will not be possible any more if the currency is not `SATS`. Maybe a new column needs to be added for `amountSats` which is updated each time a payment is made

Reply:
then you need transactions list.

Reply:
Right. For now I'll move this to a new issue as it is not a priority right now: https://github.com/getAlby/ZapPlanner/issues/75